resources/zib/zib-FunctionalOrMentalStatus.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-FunctionalOrMentalStatus" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-FunctionalOrMentalStatus" />
+  <name value="ZibFunctionalOrMentalStatus" />
+  <title value="zib FunctionalOrMentalStatus" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="The functional or mental status provides insight into the patient’s functional and mental limitations." />
+  <purpose value="This Observation resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) FunctionalOrMentalStatus v3.2 (2020)](https://zibs.nl/wiki/FunctionalOrMentalStatus-v3.2(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-functionalormentalstatus-v3.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/FunctionalOrMentalStatus-v3.2(2020EN)" />
+    <name value="zib FunctionalOrMentalStatus-v3.2(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Observation" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Observation" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation">
+      <path value="Observation" />
+      <short value="FunctionalOrMentalStatus" />
+      <comment value="The data element MedicalDevice with id NL-CM:4.26.5 is not modulated in this profile because, the reference is modulated the other way around. The resource DeviceUseStatement has an FHIR element reasonReference that has a reference to an observation resource. Because of this reference, it is not necessary to include a MedicalDevice extension in this profile." />
```

Comment:
'Modulated' klopt niet in deze zin. Bovendien is dit een erg lange zin om te zeggen wat je wil zeggen. Volgens mij kan het compacter:

> The reference to zib-MedicalDevice in concept 'MedicalDevice' (NL-CM:4.26.5), is reversed in FHIR. The relation is mapped in zib-MedicalDevice on `DeviceUseStatement.reasonReference`.

Heb je ergens bij zib-MedicalDevice (Ardon doet die) duidelijk gemaakt dat deze mapping in dat profiel moet komen? Of je moet hem zelf alvast toevoegen

Reply:
Fixt